static/usage/breadcrumbs/collapsing-items/items-before-after/vue.md

```diff
@@ -0,0 +1,54 @@
+```html
+<template>
+  <ion-content>
+    <ion-label>Before Collapse = 2</ion-label>
```

Comment:
Minor (and optional), but this is not the correct usage of `ion-label`. These should only be used in combination with components such as `ion-item` and `ion-input`: https://ionicframework.com/docs/api/label

We can probably just use a `div` or a heading element instead. Same goes for the other instances of `ion-label`.